opengrok-indexer/src/main/java/org/opengrok/indexer/history/HistoryEntry.java

```diff
@@ -139,6 +143,10 @@ public String getRevision() {
         return revision;
     }
 
+    public String getDisplayRevision() {
```

Comment:
I think this needs to have corresponding setter so that the display revision is serialized to file history cache. When I ran a test with this code, only the full revision was serialized. This means the webapp will not be able to fetch the display revision if history cache is on.

Reply:
There should probably be a test for the serialization.